aspnetcore/signalr/version-differences.md

```diff
@@ -67,6 +69,10 @@ ASP.NET Core SignalR now supports [streaming data](xref:signalr/streaming) from
 
 The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.
 
+### PersistentConnection Removal
```

Comment:
Keep Removal and lowercase it or omit it, your choice. It still exists in 4.x, so I think omitting "Removal" makes sense.  We didn't really "remove" it from Core, we just never added it.
```suggestion
### PersistentConnection
```